composeApp/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,65 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <string name="type_here">Type here…</string>
+    <string name="streamline_your_day_with">Stroomlijn je dag met</string>
+    <string name="welcome_back">Welkom Terug!</string>
+    <string name="login_welcome_text">Fijn om je terug te zien</string>
+    <string name="forgot_password">Wachtwoord vergeten?</string>
+    <string name="privacy_policy">Privacybeleid</string>
+    <string name="vehicle">Jouw Voertuig</string>
+    <string name="laadpaal">Laadpaal %1$s</string>
+    <string name="_28_september">28 September</string>
+    <string name="start">START</string>
```

Comment:
Don't capitalize string resources. This should be done in the presentation layer (your view / composable)